types/ya-disk/ya-disk-tests.ts

```diff
@@ -0,0 +1,78 @@
+import YaDisk = require('ya-disk');
+
+declare let disk: Promise<YaDisk.Disk>;
+declare let link: Promise<YaDisk.Link>;
+declare let file: Promise<YaDisk.Resource>;
+declare let operation: Promise<YaDisk.Operation>;
```

Comment:
```suggestion
let disk: Promise<YaDisk.Disk>;
let link: Promise<YaDisk.Link>;
let file: Promise<YaDisk.Resource>;
let operation: Promise<YaDisk.Operation>;
```
those are actual declaration for later assignments,